core/src/main/java/com/bumble/appyx/core/integration/NodeHost.kt

```diff
@@ -29,9 +30,8 @@ fun <N : Node> NodeHost(
     factory: NodeFactory<N>
 ) {
     val node by rememberNode(factory, customisations)
-    DisposableEffect(integrationPoint) {
-        integrationPoint.attach(node)
-        onDispose { integrationPoint.detach() }
+    LaunchedEffect(integrationPoint, node) {
+        node.integrationPoint = integrationPoint
```

Comment:
Does this need to be in a launched effect?

Reply:
yes, because otherwise it will be called on every recomposition

Reply:
Good point